graphistry/plotter.py

```diff
@@ -2,9 +2,13 @@
 from .compute.ComputeMixin import ComputeMixin 
 from .gremlin import GremlinMixin, CosmosMixin, NeptuneMixin
 from .layouts import LayoutsMixin
-from .feature_utils import FeatureMixin, has_min_dependancy as has_featurize  # type: ignore
-from .dgl_utils import DGLGraphMixin, has_dependancy as has_dgl  # type: ignore
-from .umap_utils import UMAPMixin, has_dependancy as has_umap  # type: ignore
+from .feature_utils import FeatureMixin, lazy_import_has_min_dependancy  # type: ignore
+from .dgl_utils import DGLGraphMixin, lazy_dgl_import_has_dependency  # type: ignore
+from .umap_utils import UMAPMixin, lazy_umap_import_has_dependancy  # type: ignore
+
+has_featurize, _ = lazy_import_has_min_dependancy()
```

Comment:
@silkspace  doesn't this top level execution cancel out the benefits of lazy import?